src/main/resources/templates/db_scripts/schema.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Keep schema empty to let RMB to create schema for a tenant. 
If no schema.json provided, then RMB does not create a tenant-specific schema.